s3tbx-landsat-reader/src/main/java/org/esa/s3tbx/dataio/landsat/geotiff/LandsatGeotiffReader.java

```diff
@@ -61,6 +61,7 @@
 
     private static final String RADIANCE_UNITS = "W/(m^2*sr*µm)";
     private static final String REFLECTANCE_UNITS = "dl";
+    private static final String DEGREE_UNITS = "0.01 deg";
```

Comment:
Instead of adding 0.01 to the unit we should use this value as scaling value. To immediately provide usable angle values.